### PR TITLE
fix: Downgrade Pythia8 to v10 from v12

### PR DESCRIPTION
**Pull request by paulgessinger**
Fixes reproducibility issue.

### CODE REVIEW

File: .github/workflows/builds.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ env:
 jobs:
   linux_ubuntu:
     runs-on: ubuntu-latest
-    container: ghcr.io/acts-project/ubuntu2404:52
+    container: ghcr.io/acts-project/ubuntu2404:sha-6e9b223
```
Comment (andiwand) [inline .github/workflows/builds.yml:27]:
```suggestion
    container: ghcr.io/acts-project/ubuntu2404:53
```

File: .github/workflows/builds.yml [hunk 2]
```diff
@@ -122,7 +122,7 @@ jobs:
 
   linux_examples_test:
     runs-on: ubuntu-latest
-    container: ghcr.io/acts-project/ubuntu2404:52
+    container: ghcr.io/acts-project/ubuntu2404:sha-6e9b223
```
Comment (andiwand) [inline .github/workflows/builds.yml:125]:
```suggestion
    container: ghcr.io/acts-project/ubuntu2404:53
```

File: .github/workflows/builds.yml [hunk 3]
```diff
@@ -159,7 +159,7 @@ jobs:
 
   linux_physmon:
     runs-on: ubuntu-latest
-    container: ghcr.io/acts-project/ubuntu2404:52
+    container: ghcr.io/acts-project/ubuntu2404:sha-6e9b223
```
Comment (andiwand) [inline .github/workflows/builds.yml:162]:
```suggestion
    container: ghcr.io/acts-project/ubuntu2404:53
```

**Comment by paulgessinger** [comment]
@andiwand I made the tag 53 now, and I'll just bump all CI images to that version to keep it consistent.

**Review by benjaminhuth — APPROVED** [review]

**Review by benjaminhuth — APPROVED** [review]

File: .github/workflows/checks.yml [hunk 4]
```diff
@@ -15,7 +15,7 @@ concurrency:
 jobs:
   format:
     runs-on: ubuntu-latest
-    container: ghcr.io/acts-project/format14:51
+    container: ghcr.io/acts-project/format14:53
```
Comment (andiwand) [inline .github/workflows/checks.yml:18]:
```suggestion
    container: ghcr.io/acts-project/format14:51
```

**Review by benjaminhuth — APPROVED** [review]

**Comment by paulgessinger** [comment]
And now it sporadically failed again. 🙁

**Review by andiwand — APPROVED** [review]
My bad. I think you need to force merge since I pushed already

**Comment by acts-project-service** [comment]
## :white_check_mark: Athena integration test results [6821b0410b286ac066bb240401725ecdd0f9e9b9]
### :white_check_mark: All tests successful

|status|job|report|
|------|---|------|
|:green_circle: | [report_pull_request](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243148)||
|:green_circle: | [run_art_test: test_data18_13TeV_1000evt](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243147)||
|:green_circle: | [run_art_test: test_ttbarPU40_reco](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243146)||
|:green_circle: | [test_ActsDumpGeometryIdentifiers](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243145)||
|:green_circle: | [test_ActsCheckObjectCountsWorkflow](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243144)||
|:green_circle: | [test_ActsEFTrackFit](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243143)||
|:green_circle: | [test_ActsPersistifySeeds](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243142)||
|:green_circle: | [test_ActsBenchmarkWithSpot](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243141)||
|:green_circle: | [test_ActsAnalogueClustering](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243140)||
|:green_circle: | [test_ActsWorkflowHeavyIons](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243139)||
|:green_circle: | [test_ActsWorkflowFastTracking](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243138)||
|:green_circle: | [test_ActsWorkflowCached](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243137)||
|:green_circle: | [test_ActsWorkflow](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243136)||
|:green_circle: | [test_ActsValidateAmbiguityResolution](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243135)||
|:green_circle: | [test_ActsValidateResolvedTracks](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243134)||
|:green_circle: | [test_ActsValidateTracks](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243133)||
|:green_circle: | [test_ActsValidateActsCoreSpacePoints](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243132)||
|:green_circle: | [test_ActsValidateActsSpacePoints](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243131)||
|:green_circle: | [test_ActsValidateSeeds](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243130)||
|:green_circle: | [test_ActsValidateOrthogonalSeeds](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243129)||
|:green_circle: | [test_ActsValidateClusters](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243128)||
|:green_circle: | [test_ActsPersistifyEDM](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243127)||
|:green_circle: | [test_ActsGSFRefitting](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243126)||
|:green_circle: | [test_ActsKfRefitting](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243125)||
|:green_circle: | [test_ActsExtrapolationAlgTest](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243124)||
|:green_circle: | [test_ActsITkTest](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243123)||
|:green_circle: | [run_workflow_tests_run4_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243122)||
|:green_circle: | [run_workflow_tests_run2_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243121)||
|:green_circle: | [run_workflow_tests_run2_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243120)||
|:green_circle: | [run_workflow_tests_run3_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243119)||
|:green_circle: | [run_workflow_tests_run3_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243118)||
|:green_circle: | [run_unit_tests](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41243117)||

**Pull request merged**